v2/dtos/deviceprofile.go

```diff
@@ -24,6 +27,20 @@ type DeviceProfile struct {
 	CoreCommands       []Command         `json:"coreCommands,omitempty" yaml:"coreCommands,omitempty" validate:"dive"`
 }
 
+// ToCommandModels transforms the Command DTOs to the Command models
```

Comment:
FYI, the comment doesn't match the function....

Reply:
sorry, we will correct it